cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -293,8 +293,12 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) errors.AutoscalerError
 		// Update status information when the loop is done (regardless of reason)
 		if autoscalingContext.WriteStatusConfigMap {
 			status := a.clusterStateRegistry.GetStatus(currentTime)
-			utils.WriteStatusConfigMap(autoscalingContext.ClientSet, autoscalingContext.ConfigNamespace,
+			_, writeErr := utils.WriteStatusConfigMap(autoscalingContext.ClientSet, autoscalingContext.ConfigNamespace,
 				status.GetReadableString(), a.AutoscalingContext.LogRecorder)
+			// If no other errors occurred during the run and the configmap write failed, set the typedErr
+			if typedErr == nil && writeErr != nil {
+				typedErr = errors.NewAutoscalerError(errors.ApiCallError, "%v", writeErr)
```

Comment:
Perhaps specifically on timeouts errors?
Writing that ConfigMap often fails on large clusters (where the status might exceed 1MiB). ie. seeing right now:
```
Failed to write status configmap: ConfigMap "cluster-autoscaler-status" is invalid: []: Too long: must have at most 1048576 characters
```
which is currently harmless

Reply:
good call